File: SpringBootJpa/src/main/java/com/example/simple/web/TimeController.java
package com.example.simple.web;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.time.LocalDateTime;

@RestController
@RequestMapping("date")
public class TimeController {

	@GetMapping
	public LocalDateTime getLocalDateTime() {
		return LocalDateTime.now();
	}
}

File: SpringBootJpa/src/main/kotlin/com/example/kotlin/simple/web/TimeController.kt
package com.example.kotlin.simple.web

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

import java.time.LocalDateTime

@RestController
@RequestMapping("date")
class TimeController {

    val localDateTime: LocalDateTime

    @GetMapping
    get() = LocalDateTime.now()
}
